services/mgmt/keyvault/Cargo.toml

```diff
@@ -25,23 +25,15 @@ tokio = { version = "1.0", features = ["macros"] }
 env_logger = "0.9"
 
 [package.metadata.docs.rs]
-features = ["no-default-tag", "package-2021-10", "package-preview-2021-06", "package-preview-2021-04", "package-preview-2021-04-full"]
+all-features = true
 
 [features]
 default = ["package-preview-2021-11", "enable_reqwest"]
 enable_reqwest = ["azure_core/enable_reqwest"]
 enable_reqwest_rustls = ["azure_core/enable_reqwest_rustls"]
 no-default-tag = []
+"profile-hybrid-2020-09-01" = []
 "package-preview-2021-11" = []
-"package-2021-10" = []
```

Comment:
This should probably be the default. It is the most recent stable. It may be useful to have a `limit_preview = 2` option so that there are stable tags.